xbmc/windowing/linux/OSScreenSaverFreedesktop.h

```diff
@@ -0,0 +1,49 @@
+/*
```

Comment:
Yes, I think it would be good to also use this on X11 since it is integrated into the desktop better than the XScreensaver API or moving the mouse pointer around. That's why I put it in an extra folder.